src/current/v22.2/changefeed-messages.md

```diff
@@ -134,26 +136,61 @@ Note that the original CockroachDB column definition is also included in the sch
 
 ### Schema changes with column backfill
 
-When schema changes with column backfill (e.g., adding a column with a default, adding a computed column, adding a `NOT NULL` column, dropping a column) are made to watched rows, the changefeed will emit some duplicates during the backfill. When it finishes, CockroachDB outputs all watched rows using the new schema. When using Avro, rows that have been backfilled by a schema change are always re-emitted.
+When schema changes with column backfill (e.g., adding a column with a default, adding a computed column, adding a `NOT NULL` column, dropping a column) are made to watched rows, CockroachDB emits a copy of the table using the new schema. When using Avro, rows that have been backfilled by a schema change are always re-emitted.
```

Comment:
nit: Only stored computed columns will trigger a backfill.

Reply:
>When using Avro, rows that have been backfilled by a schema change are always re-emitted.
Changefeed logic around backfills doesn't differ between Avro and json. Avro messages include a reference to the the table schema, so if you're de-duplicating messages on the other end the behavior might look a bit different?

Reply:
Updated re the first point, and removed that Avro sentence. It seems like that has been there a while and never updated.